src/styles/footer.scss

```diff
@@ -0,0 +1,26 @@
+.footer {
```

Comment:
Should we add styling so that the footer is always at the bottom irrespective of the content height?

Reply:
good point, will update!